src/services/ManageUser.ts

```diff
@@ -0,0 +1,17 @@
+import { FacilitatorListParam } from '@/utils/Interfaces';
+import { post } from './RestClient';
+
+export const getFacilitorList = async ({
+  limit,
+  page,
+  filters,
+}: FacilitatorListParam): Promise<any> => {
+  const apiUrl: string = `${process.env.NEXT_PUBLIC_BASE_URL}/list`;
+  try {
+    const response = await post(apiUrl, { limit, page, filters });
+    return response?.data?.result;
+  } catch (error) {
```

Comment:
Add error toast message

Reply:
this is service, we dont need toast msg here.